src/domain/dataset-update-flow/src/entities/schedule/schedule_type.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+/// Represents dataset update settings
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ScheduleType {
```

Comment:
Rename to `Schedule`? As it's not just the type, but the schedule itself.

Reply:
This was one of the draft names as well. I have some doubts on whether having `Schedule` (value object) and `UpdateSchedule` (aggregate) is distintcitve enough.